theme/highcontrast.less

```diff
@@ -186,6 +199,7 @@
     }
     svg.blocklySvg {
         background-color: @HCblocklySvgColor !important;
+        background: @HCblocklySvgColor !important;
```

Comment:
```suggestion
        background: @HCblocklySvgColor !important;
```
setting background will set background-color as well / is shorthand for setting all the background- props

Reply:
oooo cool ty joey!